.travis.yml

```diff
@@ -100,6 +101,11 @@ before_script:
 script:
   - if [[ $TRAVIS_OS_NAME == "linux" ]] && [[ $TRAVIS_RUST_VERSION == "stable" ]]; then
       rustup component add rustfmt && cargo fmt --all -- --check;
+      scripts/generate-bindings.sh && git diff --exit-code HEAD;
+    fi
+  - if [[ $TRAVIS_OS_NAME == "osx" ]] && [[ $TRAVIS_RUST_VERSION == "stable" ]]; then
+      rustup component add rustfmt && cargo fmt --all -- --check;
```

Comment:
Redundant check.

Reply:
@BusyJay So how do you know if the person using mac has run this script?

Reply:
ignore this, I didn't see you killing the mac